projects/media-viewer/src/lib/toolbar/toolbar.component.html

```diff
@@ -0,0 +1,7 @@
+<button (click)="rotate('LEFT')">rotate left</button>
+<button (click)="rotate('RIGHT')">rotate right</button>
```

Comment:
why not use +90/-90 so that we don't have check for direction later and can do the rotation direclty